lib/bolt/config.rb

```diff
@@ -32,16 +32,19 @@ def self.default
     end
 
     def self.from_project(project, overrides = {})
+      logs = []
       conf = if project.project_file == project.config_file
                project.data
              else
-               Bolt::Util.read_optional_yaml_hash(project.config_file, 'config')
+               c = Bolt::Util.read_optional_yaml_hash(project.config_file, 'config')
+               logs << { debug: "Loaded configuration from #{project.config_file}" }
```

Comment:
```suggestion
               logs << { debug: "Loaded configuration from #{project.config_file}" } if c.any?
```

This will log that config was loaded from `bolt.yaml` if `bolt-project.yaml` is empty.

Reply:
This is tricky because it's impossible to distinguish just from the data if we loaded an empty file or no file at all. Added a check for if the file exists.